### ISSUE TITLE
Missing scripts in default ANTSPATH

### ISSUE THREAD

**Issue by antotorov** [ANTsX/ANTs#849]
**Describe the bug**
The scripts (e.g. `antsRegistrationSyN.sh`) is not located at `${ANSPATH}`

**To Reproduce**
1. Follow the installation instructions for compiling on MACOS
2. Define ANSPATH and update the PATH variable as described there after the `make`, i.e.
```
export ANTSPATH=${HOME}/bin/ants/bin/
export PATH=${ANTSPATH}:$PATH
```
3. type `antsRegistrationSyN.sh` in the current shell session.
4. The error is 
> command not found

**Expected behavior**
Expected to show help/usage for `antsRegistrationSyN.sh`

**Desktop (please complete the following information):**
 - OS: MACOS Mojave v10.14.6

**Additional context**
The problem seems to be that the script `antsRegistrationSyN.sh` is not in the ants/bin folder. It is, instead, in the Scripts folder (available on github). My current fix is simply to copy this folder into ants/, so I have ants/bin and ants/Scripts, and add ants/Scripts to my PATH variable:

`export PATH=~/bin/ants/Scripts:$PATH`

**Comment by gdevenyi** [comment]
Fix in #841

**Comment by ntustison** [comment]
Also, [this](https://github.com/ANTsX/ANTs/wiki/Compiling-ANTs-on-Linux-and-Mac-OS#copy-scripts).
